be/src/util/stack_trace_mutex.cpp

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <array>
```

Comment:
Why add this?

Reply:
gcc 12 will complain it without this line of code